pkg/kubelet/network/kubenet/kubenet_linux.go

```diff
@@ -72,6 +76,19 @@ func (plugin *kubenetNetworkPlugin) Init(host network.Host) error {
 		glog.Warningf("Failed to find default bridge MTU: %v", err)
 	}
 
+	// Set bridge-nf-call-iptables=1 to maintain compatibility with older
+	// kubernetes versions to ensure the iptables-based kube proxy functions
+	// correctly.  Other plugins are responsible for setting this correctly
+	// depending on whether or not they connect containers to Linux bridges
+	// or use some other mechanism (ie, SDN vswitch).
+
+	// Ensure the netfilter module is loaded on kernel >= 3.18; previously
+	// it was built-in.
+	utilexec.New().Command("modprobe", "br-netfilter").CombinedOutput()
```

Comment:
Stylistic nit, I like explicit `_, _` assignment + comment when errors are being ignored. Doesn't have to change though.